server/plugin/template.go

```diff
@@ -303,9 +303,9 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 
 	template.Must(masterTemplate.New("pullRequestReviewEvent").Funcs(funcMap).Parse(`
 {{template "repo" .GetRepo}} {{template "user" .GetSender}}
-{{- if eq .GetReview.GetState "APPROVED"}} approved
-{{- else if eq .GetReview.GetState "COMMENTED"}} commented on
-{{- else if eq .GetReview.GetState "CHANGES_REQUESTED"}} requested changes on
```

Comment:
Did GitHub document the changes anywhere?

Reply:
@hanzei I was getting the state in small case and in the code as well they have been changed to lower case in the using toLower. Let me try to find the GitHub documentation for this.

Reply:
@hanzei Can't find a documentation but here is a screenshot of the webhook payload
![Screenshot from 2024-06-21 12-33-38](https://github.com/mattermost/mattermost-plugin-github/assets/90389917/ad2cbd37-49b4-42a8-8b98-3817f850026b)